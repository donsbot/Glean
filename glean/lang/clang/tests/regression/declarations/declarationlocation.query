query: |
    { N, L } where cxx1.DeclarationLocationName { _, N, L }
transform: [gensort, []]
