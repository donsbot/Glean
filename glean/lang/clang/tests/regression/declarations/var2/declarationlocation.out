[
  "@generated",
  {
    "key": {
      "tuplefield0": {
        "file": { "key": "test.cpp" },
        "lineBegin": 9,
        "columnBegin": 1,
        "lineEnd": 13,
        "columnEnd": 1
      },
      "tuplefield1": "T"
    }
  },
  {
    "key": {
      "tuplefield0": {
        "file": { "key": "test.cpp" },
        "lineBegin": 10,
        "columnBegin": 3,
        "lineEnd": 10,
        "columnEnd": 8
      },
      "tuplefield1": "m1"
    }
  },
  {
    "key": {
      "tuplefield0": {
        "file": { "key": "test.cpp" },
        "lineBegin": 11,
        "columnBegin": 3,
        "lineEnd": 11,
        "columnEnd": 12
      },
      "tuplefield1": "m2"
    }
  },
  {
    "key": {
      "tuplefield0": {
        "file": { "key": "test.cpp" },
        "lineBegin": 12,
        "columnBegin": 3,
        "lineEnd": 12,
        "columnEnd": 16
      },
      "tuplefield1": "m3"
    }
  }
]